raliat/addition.py

```diff
@@ -0,0 +1,7 @@
+def sum_numbers(num1, num2):
+    sum = num1 + num2
+    print("Sum:", sum)
+    return sum
+result = sum_numbers(20, 20)
+
```

Comment:
`sum` is a key word in python can you please change it to another name. 

Then also you don't need the print("Sum:", sum) inside what you can do is this 

```
def sum_numbers(num1, num2):
    sum_nums = num1 + num2
    return sum_nums 
result = sum_numbers(20, 20)

print(result)
```